jwt.go

```diff
@@ -31,6 +31,7 @@ type Config struct {
 	OpaUrl             string
 	OpaAllowField      string
 	OpaBody            bool
+	OpaDebugMode       bool
```

Comment:
plz use the same indentation as thr rest of the file

Reply:
As far as I can tell, the formatting looks correct....